### DISCUSSION TITLE
Anchor integration strength

### DISCUSSION THREAD

**Discussion by rLannes** [stuart-lab/signac#1074] [Q&A] [answered]
Hi, I have multiple scATAC seq of sorted malignant cell from different patient.

I try to integrate them, but for me, it's over Integrating my data. 
Could it be because they are all from the same cell type?
Do you have any advice to control the strength of the integration?

Thank you for your help!

**Comment by timoast** [comment, marked as answer]
> Could it be because they are all from the same cell type?

If everything is the same cell type, I'm not sure what the goal is in running the integration.

You can make the anchor finding more stringent by adjusting the `k.anchor` parameter

**Reply by rLannes** [reply]
Thank you for your answer,
 Those patient have different subtype of a cancers.
 We are trying to characterized those subtypes.

Just to be certain, does increasing k.anchor make the integration stronger?

**Reply by timoast** [reply]
Increasing `k.anchor` will increase the number of anchors found, and enable pairs of cells that are not as "close together" in the low-dimensional space to be identified as pairs of anchor cells.
